<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>TopPIC Suite</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="">
<meta name="author" content="xwliu" >

<link href="assets/css/bootstrap.css" rel="stylesheet">
<link href="style.css" rel="stylesheet">
<link href="assets/css/bootstrap-responsive.css" rel="stylesheet">
<link rel="shortcut icon" href="toppic.ico">
</head>

<body style="overflow-y:scroll">

<div class="container">

  <div class="masthead">
    <h2 class="muted"><font color=#0066DD>TopPIC Suite</font></h2>
    <div class="navbar">
      <div class="navbar-inner">
        <div class="container">
          <ul class="nav">
            <li><a href="index.html">Summary</a></li>
            <li><a href="register.html">Downloads</a></li>
            <li><a href="manual.html">Manual</a></li>
            <li><a href="tutorial.html">Tutorial</a></li>
            <li><a href="publications.html">Publications</a></li>
            <li><a href="contact.html">Contact</a></li>
          </ul>
        </div>
      </div>
    </div><!-- /.navbar -->
  </div><!-- /.masthead -->

  <div class="row-fluid" style="font-size:16px">

    <div class="span-12">

      <h3>1. Overview</h3>
      In this tutorial, we use TopPIC suite to analyze a top-down MS/MS data set
      of <i>Salmonella typhimurium</i> on a computer with a Windows operating
      system. <b>Annotated proteoform spectrum-matches (PrSMs) identified from the data set can be browsed 
        <a href="data_files/st_ms2_prsm_cutoff_html/proteins.html">here</a></b>.

      <h3>2. Folders</h3>
      <p>Create the folders below for software packages and data sets
      used in this tutorial. 
      </p>
      <ol>
        <li>Create a new folder named 'toppic_tutorial' on the C: drive of your system. 
        <li>Create a new subfolder named 'pwid' in the folder C:\toppic_tutorial\ for 
        the software package ProteoWizard. 
        <li>Create a new subfolder named 'toppic' in the folder C:\toppic_tutorial\ for 
        the software TopPIC suite. 
        <li>Create a new subfolder named 'data' in the folder C:\toppic_tutorial\ for 
        data files. 
        <!--
        <li>Create a new subfolder named 'results' in the folder C:\toppic_tutorial\ for saving
        data files. -->
      </ol>
      <p>The resulting folder structure is shown in the screenshot below.</p>
      <br>
      <p>
      <img src="./images/folder_1.png" height="200" width="600" style="border:
      #00008B 1px solid;"/>
      </p>


      <h3>3. Date set</h3>

      <h4>3.1 Top-down MS/MS date set</h4>
      <p>
      A top-down MS/MS data set of <i>S. typhimurium</i> is used in the
      tutorial. The protein extract of <i>S. typhimurium</i> were reduced with
      dithiothreitol and alkylated with iodoacetamide.  The protein mixture was
      separated with an HPLC system coupled to an LTQ-Orbitrap mass spectrometer (Thermo
      Fisher Scientific). MS and MS/MS spectra were collected at a resolution of
      60,000 and 30,000, respectively.  The data set contains 1,419
      collision-induced dissociation (CID) top-down MS/MS spectra.  
      </p>
      <p>Click <a href="./data_files/st.raw">here</a>
      <!-- <p>Click <a
           href="filedownload.php?filename=./data_files/st.raw">here</a> -->
      to download the data set
      and save it in the folder C:\toppic_tutorial\data\.</p>

      <h4>3.2 Protein sequence database</h4>
      <p>The <i>S. typhimurium</i> <a
        href="http://www.uniprot.org/uniprot/?query=proteome:UP000001014%20reviewed:yes">proteome
        database</a> was downloaded from the UniProt database on November 17, 2017. It
      contains 1,795 protein sequences. 
      </p>

      <p>Click <a href="filedownload.php?filename=./data_files/uniprot-st.fasta">here</a>
      to download the protein database and save it in the folder
      C:\toppic_tutorial\data\.</p>

      <p>The folder C:\toppic_tutorial\data\ is shown in the screenshot below.</p>
      <br>
      <img src="./images/folder_2.png" height="200" width="600" 
      style="border: #00008B 1px solid;"/></p>
      </p>

      <h3>4. Software tools</h3>

      <h4>4.1 Msconvert</h4>
      <p>
      Msconvert is a software tool in <B>ProteoWizard</B> that converts raw
      files into various spectrum file formats. 

      <h4>4.1.1 Required software packages to install ProteoWizard</font></h4>
      You must install the following three software packages to use ProteoWizard
      <ol>
        <li>
        <a href="https://www.microsoft.com/en-us/download/details.aspx?id=5582">Microsoft
          Visual C++ 2008 SP1 (x86) Redistributable Package</a></li>
        <li>
        <a href="https://www.microsoft.com/en-us/download/details.aspx?id=22">
          Microsoft .NET Framework 3.5 SP1</a></li>
        <li>
        <a href="https://www.microsoft.com/en-us/download/details.aspx?id=17851">
          Microsoft .NET Framework 4.0</a></li>
      </ol>

      <h4>4.1.2 Download ProteoWizard</h4>
      <p>
      Follow the steps to below download ProteoWizard:
      </p>
      <p>
      <ol>
        <li>Go to the link  <a href = "http://proteowizard.sourceforge.net/downloads.shtml">ProteoWizard</a>.</li>
        <li>Choose the download type 'Windows 64-bit tar.bz2' for end users.</li>
        <li>Click the button "I agree to the licensing terms, download
        ProteoWizard" to download ProteoWizard and save the bz2 file in the folder
        C:\toppic_tutorial\pwid\.</li>
        <li>Extract all the files of the downloaded bz2 file to the folder
        C:\toppic_tutorial\pwiz\.</li> 
        <br>
        <p><img src="./images/pwiz_1.png" height="200" width="600" 
        style="border: #00008B 1px solid;"/>
        </p>
      </ol>
      <br>

      <h4>4.2 TopPIC suite</h4>

      <h4>4.2.1 Download TopPIC suite</h4>
      <p>
      <ol>
        <li>Go to the <a href = "./register.html">link</a> to download
      webpage of TopPIC.</li>
        <li>Choose the download type "Windows 64-bit zip file," fill out the
        registration form, and click "I accept license agreement and download
        TopPIC suite"
        to download it.</li>
        <li>Save it to the folder C:\toppic_tutorial\toppic\.</li>
        <li>Extract all the files of the downloaded zip file to the folder
        C:\toppic_tutorial\toppic\.</li> 
        <br>
        <p><img src="./images/toppic_1.png" height="200" width="600" 
        style="border: #00008B 1px solid;"/>
        </p>
      </ol>

      <h3>5 Data analysis using GUI</h3>
      <h4>5.1 File format conversion</h4>

      <p>We use <b>MSConvertGUI</b> to convert the raw file st.raw to an mzML
      file.</p>
      <ol>
        <li> Double click the executable file MSConvertGUI.exe in the folder
        C:\toppic_tutorial\pwiz.</li>
        <li> Add the file c:\toppic_tutorial\data\st.raw as an input file.</li>
        <li> Add the filter "peakPeaking vendor msLevel=1-" <b><font
            color=#FF0000">(important)</font></b>.</li>
        <li> Click to the button "Start" to perform file format conversion.</li>
      </ol>
      <p>
      The screenshot of MSConvertGUI is shown below. 
      </p>
      <p><img src="./images/msconvert_1.png" height="400" width="600" 
      style="border: #00008B 1px solid;"/>
      </p>

      <p>
      In the above file format conversion, the peak picking filter (step 3) is used to
      generate a centroided, not profile, mzML data file, which is required by the
      spectral deconvolution tool TopFD. 
      </p>

      <p>
      The resulting output mzML file is
      <pre>C:\toppic_tutorial\result\st.mzML</pre> The size of the file is about 43
      MB, which can be downloaded <a
      href="filedownload.php?filename=./data_files/st.mzML">here</a>.  The
    running time for the file format conversion is less than one minute.   
      </p>

      <h4>5.2 Mass spectral deconvolution</h4>

      <p>We use <b>topfd_gui</b> for top-down mass spectral deconvolution.</p>
      <ol>
        <li> Double click the executable file topfd_gui.exe in the folder
        C:\toppic_tutorial\toppic.</li>
        <li> Add the file c:\toppic_tutorial\data\st.mzML as an input file.</li>
        <li> Click to the button "Start" to deconvolute the file.</li>
      </ol>
      <p>
      The screenshot of topfd_gui is shown below. 
      </p>

      <p><img src="./images/topfd_1.png" height="400" width="600" 
      style="border: #00008B 1px solid;"/>
      </p>

      TopFD reports three text files. 
      <ol>
        <li>A msalign file containing deconvoluted MS1 spectra:
        <pre>C:\toppic_tutorial\data\st_ms1.msalign</pre></li>
        <li>A msalign file containing deconvoluted MS/MS spectra:
        <pre>C:\toppic_tutorial\data\st_ms2.msalign</pre></li>
        <li>A text file containing LC/MS features of MS/MS spectra:
        <pre>C:\toppic_tutorial\data\st.feature</pre></li>
      </ol>
      <p>
      The three output files can be downloaded
      <a href="filedownload.php?filename=./data_files/st-msalign.zip">here</a>.  
      The running time of TopFD is about 6 minutes. 
      </p>

      <h4>5.3 Mass spectral identification by database search</h4>
      <p>We use <b>toppic_gui</b> to search MS/MS spectra in the file
      st_ms2.msalign against the protein database uniprot-st.fasta to
      identify PrSMs. 
      </p>
      <ol>
        <li> Double click the executable file toppic_gui.exe in the folder
        C:\toppic_tutorial\toppic.</li>
        <li> Select C:\toppic_tutorial\data\uniprot-st.fasta as the protein
        database file.</li>
        <li> Select C:\toppic_tutorial\data\st_ms2.msalign as the
        mass spectrum data file. </li>
        <li> Check the checkbox "MS1 feature file" and select 
        C:\toppic_tutorial\data\st.feature as the feature file. </li>
        <li> Select "C57" as the fixed modification. </li>
        <li> Check the checkbox "Decoy database." </li>
        <li> Select "FDR" as the spectrum level cutoff type. </li>
        <li> Select "FDR" as the proteoform level cutoff type. </li>
        <li> Click to the button "Start"</li>
      </ol>
      <p>
      The screenshot of toppic_gui is shown below. 
      </p>

      <p><img src="./images/toppic_2.png" height="400" width="600" 
      style="border: #00008B 1px solid;"/>
      </p>


      <p>TopPIC reports two tab delimited text files and a collection of html
      files for identified proteoforms. 
      <ul>
        <li>A tab delimited text file containing identified PrSMs with a 1% spectrum level FDR:
        <pre>C:\toppic_tutorial\data\st_ms2.OUTPUT_TABLE</pre> 
        </li>
        <li> A tab delimited text file containing identified proteoforms and
        their best PrSMs with a 1% proteoform level FDR:
        <pre>C:\toppic_tutorial\data\st_ms2.FORM_OUTPUT_TABLE</pre> 
        </li> A folder containing html files for the
        annotation of identified PrSMs with a 1% spectrum
        level FDR: 
        <pre>C:\toppic_tutorial\data\st_ms2_prsm_cutoff_html</pre> 
        </li>
        <li>
        A folder containing html files for the
        annotation of identified PrSMs with a 1%
        proteoform level FDR: 
        <pre>C:\toppic_tutorial\data\st_ms2_proteoform_cutoff_html</pre> 
        </li>
      </ul>
      </p>

      <p> In the analysis, C57 is selected as the fixed modification because
      proteins were reduced with dithiothreitol and alkylated with
      iodoacetamide before the MS experiment. When proteins are not reduced, C0
      should be selected. A shuffled decoy database is concatenated
      to the target database to estimate spectrum level and proteoform level
      FDRs. All identified PrSMs are first filtered by a
      1% spectrum level FDR and the resulting PrSMs are reported in the
      file st_ms2.OUTPUT_TABLE. The proteoforms corresponding to the PrSMs 
      are further filtered using a 1% proteoform level FDR and
      the resulting proteoforms and their corresponding best PrSMs are reported in the file
      st_ms2.FORM_OUTPUT_TABLE. Microsoft Excel can be used to open these two files. 
      To browse the annotations in the two folders st_ms2_prsm_cutoff_html and
      st_ms2_proteoform_cutoff_html, use a web browser (Google Chrome is
      recommended) to open the file proteins.html.
      </p>

      <p>
      The output files can be downloaded
      <!-- <a href="filedownload.php?filename=./data_files/st-result.zip">here</a>.-->  
      <a href="./data_files/st-results.zip">here</a>.  
      The running time of TopPIC is about 5 minutes. 
      </p>
      

      <h3>6 Data analysis using the command line interface</h3>
      <h4>6.1 File format conversion</h4>

      <p>We use <b>msconvert</b> to convert the raw file ecoli.raw to an mzML
      file.</p>
      <h4>File locations</h4>
      <ol>
        <li>Executable file: <pre>C:\toppic_tutorial\pwid\msconvert.exe</pre></li>
        <li>Input raw file: <pre>C:\toppic_tutorial\data\st.raw</pre></li>
      </ol>

      <h4>Command</h4>
      <ol>
        <pre class="code">cd c:\toppic_tutorial\data</pre>
        <pre class="code">..\pwid\msconvert.exe filter "peakPicking true 1-" st.raw</pre>
      </ol>
      <p>In the above command, the argument filter "peakPicking true 1-" is used to
      generate a centroided, not profile, mzML data file, which is needed by the
      spectral deconvolution tool TopFD. 
      </p>
      <!--
      <p>The screenshot below shows the console output of the above command.</li>
      <p><img src="" height="250" width="500" border="1.5"/></p>
      -->

      <h4>Output file</h4>
      <p>
      The resulting output mzML file is
      <pre>C:\toppic_tutorial\result\st.mzML</pre> The size of the file is about 43
      MB, which can be downloaded <a
      href="filedownload.php?filename=./data_files/st.mzML">here</a>.  The
    running time for the file format conversion is less than one minute.   
      </p>


      <h4>6.2 Mass spectral deconvolution</h4>

      <p>We use <b>TopFD</b> for top-down mass spectral deconvolution.</p>

      <h4>File locations</h4>
      <ol>
        <li>Executable file: <pre>C:\toppic_tutorial\toppic\topfd.exe</pre></li>
        <li>Input mzML file: <pre>C:\toppic_tutorial\data\st.mzML</pre></li>
      </ol>
      <h4>Command</h4>
      <pre class="code">cd c:\toppic_tutorial\data</pre>
      <pre class="code">..\toppic\topfd st.mzML</pre>
      <!--
      <p>The screenshot below shows the console output of the above command.</li>
      <p><img src="" height="250" width="500" border="1.5"/></p>
      -->

      <h4>Output files</h4>
      TopFD reports three text files. 
      <ol>
        <li>A msalign file containing deconvoluted MS1 spectra:
        <pre>C:\toppic_tutorial\data\st_ms1.msalign</pre></li>
        <li>A msalign file containing deconvoluted MS/MS spectra:
        <pre>C:\toppic_tutorial\data\st_ms2.msalign</pre></li>
        <li>A text file containing LC/MS features of MS/MS spectra:
        <pre>C:\toppic_tutorial\data\st.feature</pre></li>
      </ol>
      <p>
      The three output files can be downloaded
      <a href="filedownload.php?filename=./data_files/st-msalign.zip">here</a>.  
      The running time of TopFD is about 6 minutes. 
      </p>

      <h4>6.3 Mass spectral identification by database search</font></h4>
      <p>We use <b>TopPIC</b> to search MS/MS spectra in the file st_ms2.msalign
      against the protein database uniprot-st.fasta to identify PrSMs.  The
      feature file st.feature is used to map identified PrSMs to proteoforms.  
      </p>
      <h4>File locations</h4>
      <ol>
        <li>Executable file: <pre>C:\toppic_tutorial\toppic\toppic.exe</pre></li>
        <li>Input protein database file:
        <pre>C:\toppic_tutorial\data\unprot-st.fasta</pre></li>
        <li>Input MS/MS msalign file: <pre>C:\toppic_tutorial\data\st_ms2.msalign</pre></li>
        <li>Input feature file: <pre>C:\toppic_tutorial\data\st.feature</pre></li>
      </ol>
      <h4>Commands</h4>
      <pre class="code">cd c:\toppic_tutorial\data</pre>
      <pre class="code">..\toppic\toppic -f C57 -d -t FDR -T FDR -x st.feature uniprot-st.fasta st_ms2.msalign</pre>
      <!--
      <p>The screenshot below shows the console output of the above command.</li>
      <p><img src="" height="250" width="500" border="1.5"/></p>
      -->
      <h4>Output files</h4>
      <p>TopPIC reports two tab delimited text files and a collection of html
      files for identified proteoforms. 
      <ul>
        <li>A tab delimited text file containing identified PrSMs with a 1% spectrum level FDR:
        <pre>C:\toppic_tutorial\data\st_ms2.OUTPUT_TABLE</pre> 
        </li>
        <li> A tab delimited text file containing identified proteoforms and
        their best PrSMs with a 1% proteoform level FDR:
        <pre>C:\toppic_tutorial\data\st_ms2.FORM_OUTPUT_TABLE</pre> 
        </li> A folder containing html files for the
        annotation of identified PrSMs with a 1% spectrum
        level FDR: 
        <pre>C:\toppic_tutorial\data\st_ms2_prsm_cutoff_html</pre> 
        </li>
        <li>
        A folder containing html files for the
        annotation of identified PrSMs with a 1%
        proteoform level FDR: 
        <pre>C:\toppic_tutorial\data\st_ms2_proteoform_cutoff_html</pre> 
        </li>
      </ul>
      </p>

      <p> In the analysis, C57 (-f C57) is selected as the fixed modification because
      proteins were reduced with dithiothreitol and alkylated with
      iodoacetamide before the MS experiment. When proteins are not reduced, C0
      should be selected. A shuffled decoy database (-d) is concatenated
      to the target database to estimate spectrum level and proteoform level
      FDRs. All identified PrSMs are first filtered by a
      1% spectrum level FDR (-t FDR) and the resulting PrSMs are reported in the
      file st_ms2.OUTPUT_TABLE. The proteoforms corresponding to the PrSMs 
      are further filtered using a 1% proteoform level FDR (-T FDR) and
      the resulting proteoforms and their corresponding best PrSMs are reported in the file
      st_ms2.FORM_OUTPUT_TABLE. The feature file st.feature is used to map
      identified PrSMs to proteoforms (-x st.feature). Microsoft Excel can be used to open these two files. 
      To browse the annotations in the two folders st_ms2_prsm_cutoff_html and
      st_ms2_proteoform_cutoff_html, use a web browser (Google Chrome is
      recommended) to open the file proteins.html.
      </p>

      <p>
      The output files can be downloaded
      <!-- <a href="filedownload.php?filename=./data_files/st-result.zip">here</a>.-->  
      <a href="./data_files/st-results.zip">here</a>.  
      The running time of TopPIC is about 5 minutes. 
      </p>
    </div>

  </div>
</div>
<script src="assets/js/jquery-1.7.1.min.js"></script>

</body>
</html>
