<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>TopPIC Suite</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="qkou" >

    <link href="assets/css/bootstrap.css" rel="stylesheet">
    <style type="text/css">
      body {
        padding-top: 20px;
        padding-bottom: 60px;
      }

      /* Custom container */
      .container {
        margin: 0 auto;
        max-width: 1000px;
      }
      .container > hr {
        margin: 60px 0;
      }

      /* Main marketing message and sign up button */
      .jumbotron {
        margin: 80px 0;
        text-align: center;
      }
      .jumbotron h1 {
        font-size: 100px;
        line-height: 1;
      }
      .jumbotron .lead {
        font-size: 24px;
        line-height: 1.25;
      }
      .jumbotron .btn {
        font-size: 21px;
        padding: 14px 24px;
      }

      /* Supporting marketing content */
      .marketing {
        margin: 60px 0;
      }
      .marketing p + h4 {
        margin-top: 28px;
      }


      /* Customize the navbar links to be fill the entire space of the .navbar */
      .navbar .navbar-inner {
        padding: 0;
      }
      .navbar .nav {
        margin: 0;
        display: table;
        width: 100%;
      }
      .navbar .nav li {
        display: table-cell;
        width: 1%;
        float: none;
      }
      .navbar .nav li a {
        font-weight: bold;
        text-align: center;
        border-left: 1px solid rgba(255,255,255,.75);
        border-right: 1px solid rgba(0,0,0,.1);
      }
      .navbar .nav li:first-child a {
        border-left: 0;
        border-radius: 3px 0 0 3px;
      }
      .navbar .nav li:last-child a {
        border-right: 0;
        border-radius: 0 3px 3px 0;
      }
      
      .flex-video {
		position: relative;
		padding-top: 25px;
		padding-bottom: 67.5%;
		height: 0;
		margin-bottom: 16px;
		overflow: hidden;
	  }
 
	  .flex-video.widescreen { padding-bottom: 57.25%; }
	  .flex-video.vimeo { padding-top: 0; }
 
	  .flex-video iframe,
	  .flex-video object,
	  .flex-video embed {
		position: absolute;
		top: 0;
		left: 0;
		width: 100%;
		height: 100%;
	  }
	  
	  .code {
		color: #777;  
	  }
	 
    </style>
    <link href="assets/css/bootstrap-responsive.css" rel="stylesheet">
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="toppic.ico">
    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="toppic.ico">
      <link rel="apple-touch-icon-precomposed" sizes="72x72" href="toppic.ico">
                    <link rel="apple-touch-icon-precomposed" href="toppic.ico">
                                   <link rel="shortcut icon" href="toppic.ico">
  </head>

<body style="overflow-y:scroll">

<div class="container">

  <div class="masthead">

    <h2 class="muted"><font color=#0066DD>TopPIC Suite</font></h2>
    <div class="navbar">
      <div class="navbar-inner">
        <div class="container">
          <ul class="nav">
            <li><a href="index.html">Summary</a></li>
            <li><a href="register.html">Downloads</a></li>
            <li><a href="manual.html">Manual</a></li>
            <li><a href="tutorial.html">Tutorial</a></li>
            <li><a href="publications.html">Publications</a></li>
            <li><a href="contact.html">Contact</a></li>
          </ul>
        </div>
      </div>
    </div><!-- /.navbar -->
  </div><!-- /.masthead -->

  <div class="row-fluid" style="font-size:16px">

    <div class="span-12">

      <h3><font color=#0066DD>1. Overview</font></h3>
      In this tutorial, we use TopPIC to analyze a top-down MS/MS data set
      of <i>Escherichia coli</i> on a computer with a Windows operating system.     

      <h3><font color=#0066DD>2. Folders</font></h3>
      <p>Create the folders below for saving software packages and data sets
      used in this tutorial. 
      </p>
      <ol>
        <li>Create a new folder named 'tutorial' on the C: drive of your system. 
        <li>Create a new subfolder named 'pw' in the folder C:\tutorial\ for saving
        the software package Proteowizard. 
        <li>Create a new subfolder named 'toppic' in the folder C:\tutorial\ for saving
        the software TopPIC suite. 
        <li>Create a new subfolder named 'data' in the folder C:\tutorial\ for saving
        data files. 
        <li>Create a new subfolder named 'result' in the folder C:\tutorial\ for saving
        data files. 
      </ol>
      <p>The resulting folder structure is shown in the screenshot below.</p>
      <br>
      <p>
      <img src="./images/folder/folder_1.jpg" height="200" width="1100" 
      style="border: #00008B 1px solid;"/></p>
      </p>


      <h3><font color=#0066DD>3. Date set</font></h3>

      <h4><font color=#0066DD>3.1 Top-down MS/MS date set</font></h3>
      <p>
      The top-down MS/MS data set was generated from protein extract of
      <i>Escherichia coli</i> K-12 MG1655. The protein extract was analyzed by
      an RPLC system coupled with an LTQ Orbitrap Velos mass spectrometry. MS1
      and MS/MS spectra were collected at a 60,000 resolution with the
      alternating fragmentation mode. The data set contains 2,027
      collision-induced dissociation and 2,027 electron-transfer
      dissociation top-down MS/MS spectra (see <a href="#reference">reference
        [1]</a>). 
      </p>
      <p>Click <a href="./data_files/tmp.raw">here</a> to download the data set
      and save it in the folder C:\tutorial\data\.</p>

      <h4><font color=#0066DD>3.2 Protein sequence database</font></h3>
      <p>The <i>Escherichia coli</i> K-12 MG1655 <a
        href="http://www.uniprot.org/uniprot/?query=proteome:UP000000625">proteome database</a> was downloaded
      from UniProt on September 14, 2017. It contains 4,309 protein sequences. 
      </p>


      <p>Click <a href="./data_files/uniprot_ecoli.fasta">here</a> to download
      the protein database and save it in the folder C:\tutorial\data\.</p>

      <p>The folder C:\tutorial\data\ is shown in the screenshot below.</p>
      <br>
      <img src="./images/folder/folder_2.jpg" height="200" width="1100" 
      style="border: #00008B 1px solid;"/></p>
      </p>

      <h3><font color=#0066DD>4. Software tools</font></h3>

      <h4><font color=#0066DD>4.1 Msconvert</font></h4>
      <p>
      Msconvert is a software tool in <B>ProteoWizard</B> that converts raw files into various formats. 

      <h4><font color=#0066DD>4.1.1 Required software packages to install ProteoWizard</font></h4>
      You must install the following three software packages to use ProteoWizard
      <ol>
        <li>
        <a href="https://www.microsoft.com/en-us/download/details.aspx?id=5582">Microsoft
          Visual C++ 2008 SP1 (x86) Redistributable Package</a></li>
        <li>
        <a href="https://www.microsoft.com/en-us/download/details.aspx?id=22">
          Microsoft .NET Framework 3.5 SP1</a></li>
        <li>
        <a href="https://www.microsoft.com/en-us/download/details.aspx?id=17851">
          Microsoft .NET Framework 4.0</a></li>
      </ol>

      <h4><font color=#0066DD>4.1.2 Download ProteoWizard</font></h4>
      <p>
      Follow the steps to below download ProteoWizard:
      </p>
      <p>
      <ol>
        <li>Go to the link  <a href = "http://proteowizard.sourceforge.net/downloads.shtml">ProteoWizard</a>.</li>
        <li>Choose the download type 'Windows 64-bit tar.bz2' for end users.</li>
        <!--
        <br>
        <p><img src="./images/msconvert/msconvert_1.jpg" height="200" width="1100" 
        style="border: #00008B 1px solid;"/>
        </p>
        -->
        <li>Click the button "I agree to the licensing terms, download
        ProteoWizard" to download ProteoWizard and save the bz2 file in the folder
        C:\tutorial\pw\.</li>
        <!--
        <br>
        <p><img src="./images/msconvert/msconvert_2.jpg" height="200" width="1100" 
        style="border: #00008B 1px solid;"/>
        </p>
        -->
        <li>Extract all the files of the downloaded bz2 file to the folder
        C:\tutorial\pw\.</li> 
        <br>
        <p><img src="./images/msconvert/msconvert_3.jpg" height="200" width="1100" 
        style="border: #00008B 1px solid;"/>
        </p>
      </ol>
      <br>

      <h4><font color=#0066DD>4.2 TopPIC suite</font></h4>

      <h4><font color=#0066DD>4.2.1 Download TopPIC suite</font></h3>
      <p>
      <ol>
        <li>Go to the <a href = "./register.html">link</a> to download
      webpage of TopPIC.</li>
        <li>Choose the download type "Windows 64-bit zip file," fill out the
        registration form, and click "I agree to the license agreement and download"
        to download TopPIC suite.</li>
        <!--
        <br>
        <p><img src="./images/toppic/toppic_1.jpg" height="200" width="1100" 
        style="border: #00008B 1px solid;"/>
        </p>
        -->
        <li>Save it to the folder C:\tutorial\toppic\.</li>
        <!--
        <br>
        <p><img src="./images/tmp.jpg" height="200" width="1100" 
        style="border: #00008B 1px solid;"/>
        </p>
        -->
        <li>Extract all the files of the downloaded zip file to the folder
        C:\tutorial\toppic\.</li> 
        <br>
        <p><img src="./images/tmp.jpg" height="200" width="1100" 
        style="border: #00008B 1px solid;"/>
        </p>
      </ol>

      <h3><font color=#0066DD>5 Data analysis</font></h3>
      <h4><font color=#0066DD>5.1 File format conversion</font></h4>

      <p>We use <b>msconvert</b> to convert the raw file ecoli.raw to an mzML
      file ecoli.mzML.</p>
      <h4><font color=#0066DD>5.1.1 File locations</font></h4>
      <ol>
        <li>Executable file: <pre>C:\tutorial\pw\msconvert.exe</pre></li>
        <li>Input raw file: <pre>C:\tutorial\data\ecoli.raw</pre></li>
      </ol>

      <h4><font color=#0066DD>5.1.2 Command</font></h4>
      <ol>
        <li>Launch the Command Prompt </li>

        <li>Input the following command: <br> 
        <pre class="code">c:\tutorial\pw\msconvert.exe --filter "peakPicking true 1" c:\tutorial\data\ecoli.raw -o c:\tutorial\result</pre>
        </li>
      </ol>
      <p>In the above command, the argument --filter "peakPicking true 1-" is used to
      generate a centroid, not profile, mzML data file, which is needed by the
      spectral deconvolution tool TopFD. The -o argument specifies the
      location of the output file. 
      </p>
      <p>The screenshot below shows the console output of the above command.</li>
      <p><img src="" height="250" width="500" border="1.5"/></p>

      <h4><font color=#0066DD>5.1.3 Output file</font></h4>
      The resulting output mzML file is <pre>C:\tutorial\result\ecoli.mzML</pre>
      The size of the file is ??? MB, which can be downloaded <a
      href="">here</a>.  The running time of the command is about one minute.   

      <h4><font color=#0066DD>5.2 Mass spectral deconvolution</font></h4>

      <p>We use <b>TopFD</b> to do mass spectral deconvolution, generating
      three files containing deconvoluted mass spectra and MS1 features from
      the mzML file ecoli.mzML.</p>
      <h4><font color=#0066DD>5.2.1 File locations</font></h4>
      <ol>
        <li>Executable file: <pre>C:\tutorial\toppic\topfd.exe</pre></li>
        <li>Input mzML file: <pre>C:\tutorial\result\ecoli.mzML</pre></li>
      </ol>
      <h4><font color=#0066DD>5.2.2 Command</font></h4>
      In the Command Prompt, input the following command:
      <pre>c:\tutorial\toppic\topfd c:\tutorial\result\ecoli.mzML</pre>
      <p>The screenshot below shows the console output of the above command.</li>
      <p><img src="" height="250" width="500" border="1.5"/></p>

      <h4><font color=#0066DD>5.2.3 Output file</font></h4>
      TopFD reports three text files. 
      <ol>
        <li>Msalign file containing deconvoluted MS1 spectra:
        <pre>C:\tutorial\result\ecoli_ms1.msalign</pre></li>
        <li>Msalign file containing deconvoluted MS/MS spectra:
        <pre>C:\tutorial\result\ecoli_ms2.msalign</pre></li>
        <li>Text file containing MS1 features of MS/MS spectra:
        <pre>C:\tutorial\result\ecoli.msft</pre></li>
      </ol>
      <p>
      The three output files can be downloaded <a href="">here</a>. The running time
      of the command is about ??? minutes. 
      </p>
      
      <h4><font color=#0066DD>5.3 Mass spectral identification by database search</font></h4>

      <h3><font color=#0066DD>6. Reference</font></h3>
      <p id="reference">
      1. Qiang Kou, Likun Xun, and Xiaowen Liu.
      <b>TopPIC: a software tool for top-down mass spectrometry-based proteoform identification and characterization.</b>
      Bioinformatics, 32:3495-3497, 2016. 
      <a target="_blank"
        href="http://www.ncbi.nlm.nih.gov/pubmed/27423895">Pubmed</a>.
      <p>
      
  <script src="assets/js/jquery-1.7.1.min.js"></script>

  </body>
</html>
